src/format/parser.d

```diff
@@ -2503,6 +2511,18 @@ private:
 		assert(0);
 	}
 
+	void parseShortenedFunctionBody() in(match(TokenType.FatArrow)) {
+		auto spanGuard = span!Span();
```

Comment:
Why do you not splice it?

Reply:
I changed it, not entirely sure if I know what the difference is precisely?

Reply:
Dump the IR, but long story short, the span is extended leftward in a way that fits the hierarchy.

Reply:
Also, if there is already a span that does that, then maybe it's not useful to add one more, but I don't remember what is the generated IR here.